airbyte-integrations/connectors/source-google-analytics-data-api/acceptance-test-config.yml

```diff
@@ -27,14 +23,30 @@ acceptance_tests:
       - config_path: "secrets/config.json"
         configured_catalog_path: "integration_tests/configured_catalog.json"
         ignored_fields:
-          "daily_active_users": ["uuid"]
-          "weekly_active_users": ["uuid"]
-          "four_weekly_active_users": ["uuid"]
-          "devices": ["uuid"]
-          "locations": ["uuid"]
-          "pages": ["uuid"]
-          "traffic_sources": ["uuid"]
-          "website_overview": ["uuid"]
+          daily_active_users:
+            - name: "uuid"
+              bypass_reason: "property is changing from sync to sync"
```

Comment:
Because we are using live data, yes?

Reply:
cc @ycherniaiev @igrankova 
Are we using the the live data for this connector configuration, probably you have any information?